react-app/src/components/Auth/LoginPage/index.js

```diff
@@ -1,5 +1,64 @@
 import React from 'react';
+import 'antd/dist/antd.css';
+import '../../../index.css';
+import { Form, Input, Button, Checkbox } from 'antd';
+import { UserOutlined, LockOutlined } from '@ant-design/icons';
 
-const LoginPage = () => <div>LoginPage</div>
+const LoginPage = () => {
+  
+  const onFinish = values => {
+    console.log('Received values of form: ', values);
+  };
```

Comment:
everything looks great! I'm assuming you need these console logs for later?

Reply:
we can removed them later when we have actually functionality for submit but it currently is just there to make sure the form and button work